user/pages/04.Reference/10.resource-limits-quota-usage/docs.en.md

```diff
@@ -1,19 +1,49 @@
 ---
-title: 'REST API for Quota Information'
+title: 'Resource Limits, REST API for Quota and Usage'
 published: true
 date: '26-07-2021 16:25'
 taxonomy:
     category:
         - docs
 ---
 
-## Introduction
+## Reasoning behind resource limits
 
-For many resource types like instances, VCPUs, volume storage you can check the limits and the current usage in the dashboard (Horizon) or use the standard OpenStack APIs to collect the data in some automation. But information about quota and usage of object storage (S3) is not available using standard APIs.
+The SysEleven Stack imposes resource limits per region on customer projects. This may sound a little bit bossy, given the fact that our customers have access to plans where they pay on-demand what they use. Couldn't we waive all restrictions and let customers use what they want? For the time being, we decided to stick with quota limits, because they server for the following purposes:
```

Comment:
```suggestion
The SysEleven Stack imposes resource limits per region on customer projects. This may sound a little bit bossy, given the fact that our customers have access to plans where they pay on-demand what they use. Couldn't we waive all restrictions and let customers use what they want? For the time being, we decided to stick with quota limits, because they serve for the following purposes:
```